manubot/cite/tests/test_csl_item.py

```diff
@@ -66,4 +106,3 @@ def test_csl_item_set_standard_id_note():
     note_dict = parse_csl_item_note(csl_item['note'])
     assert note_dict['original_id'] == 'original-id'
     assert note_dict['original_standard_id'] == 'doi:10.1371/journal.PPAT.1006256'
-
```

Comment:
unremove final newline

Reply:
is this a correct version? seems like a newline is there